.github/workflows/deploy.yml

```diff
@@ -34,20 +34,22 @@ jobs:
             - run: npm link ../dynacred
 
             - name: copy assets from remote
-              run: scp "${{ env.DEPLOY_USER }}@${{ env.DEPLOY_HOST }}:{bin/config.toml,api/cas.conf}" src/assets/
+              run: scp "${{ env.DEPLOY_USER }}@${{ env.DEPLOY_HOST }}:{www/assets/*}" src/assets/
 
             - run: npx ng build --prod --base-href / --aot --output-path dist
             - name: build for old login
               run: |
                   npx ng build --prod --base-href /omniledger/ --aot --output-path omniledger_dist
                   cp omniledger_dist/index.html dist/index_ol.html
-            - run: make cas
-              working-directory: webapp/cas
 
             - name: rsync built
               run: >
                   rsync --archive --delete dist/
                   "${{ env.DEPLOY_USER }}@${{ env.DEPLOY_HOST }}:www"
+            - name: remotely rebuild cas dameon
+              run: >
+                  ssh "${{ env.DEPLOY_USER }}@${{ env.DEPLOY_HOST }}"
+                  go install github.com/c4dt/omniledger/webapp/cas/cmd/cas@latest
```

Comment:
The idea behind building everything in the CI is to reduce the time to deploy (as deps are already present and built), and to avoid subtil bugs when using different golang/code version. Using `make` and moving `cas` to `dist` does the trick.

Reply:
The idea is to create a release with two directories inside, plus a README.md:
```
www/
bin/
README.md
```

I'd also remove the `config.toml`, and add a short explanation in the `README.md` that you need to provide a valid `config.toml` and `cas.toml`, if you want to use it on your server.

Then change the deploy-script to download the release, untar it, and restart the service.